### PR TITLE
style: Update shadows to new style.

### PR DESCRIPTION
**Pull request by milesj**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description

Updates our shadows to:

Large `0px 8px 28px rgba(0, 0, 0, 0.28)`
Default `0px 6px 20px rgba(0, 0, 0, 0.2)`
Small `0px 4px 16px rgba(0, 0, 0, 0.12)`

## Motivation and Context

Just part of the new design.

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/core/src/components/Lightbox/index.tsx [hunk 1]
```diff
@@ -140,7 +140,7 @@ export default class Lightbox extends React.PureComponent<LightboxProps, Lightbo
     );
 
     return (
-      <Sheet compact headerShadow noAnimation portal visible header={header} onClose={onClose}>
```
Comment (stefhatcher) [inline packages/core/src/components/Lightbox/index.tsx]: instead of the shadow, add a border bottom to the header

Comment (milesj) [reply packages/core/src/components/Lightbox/index.tsx]: Switched it to a thick border.

File: packages/core/src/components/Sheet/styles.ts [hunk 2]
```diff
@@ -147,7 +147,7 @@ export const styleSheetSheet: StyleSheet = ({ color, pattern, ui, unit }) => ({
   },
 
   headerShadow: {
-    boxShadow: ui.boxShadow,
+    borderBottom: ui.borderThick,
```
Comment (stefhatcher) [inline packages/core/src/components/Sheet/styles.ts]: why thick?

Comment (milesj) [reply packages/core/src/components/Sheet/styles.ts]: It's the same height to what the shadow was. IMO, I like it.

**Pull request merged**